otel-integration/k8s-helm/values.yaml

```diff
@@ -55,11 +55,8 @@ opentelemetry-agent:
     prometheusCR:
       enabled: true
     image:
-      # Use custom image for target allocator until
-      # the official image is updated with the "per node" strategy
-      # https://github.com/open-telemetry/opentelemetry-operator/pull/2430
-      repository: coralogixrepo/target-allocator
-      tag: sha-abe05c5
+      repository: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator
+      tag: v0.95.0
```

Comment:
Just FYI, operator is yet to have `0.96.0` release, so I'm using `0.95.0`, but the version are compatible just fine.